crates/eframe/src/epi.rs

```diff
@@ -1145,6 +1159,10 @@ pub(crate) mod backend {
         #[cfg(not(target_arch = "wasm32"))]
         pub focus: Option<bool>,
 
+        /// Set to request a user's attention to the native window. Setting this to `None` resets the attention request.
```

Comment:
This docstring is wrong I think. `None` means "keep the current attention", afaict

```suggestion
        /// Set to request a user's attention to the native window.
```